packages/xml-views-validation/src/validators/attributes/invalid-boolean-value.ts

```diff
@@ -0,0 +1,52 @@
+import { XMLAttribute } from "@xml-tools/ast";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { getPropertyByAttributeKey } from "@ui5-language-assistant/logic-utils";
+import { find, map } from "lodash";
+import { InvalidBooleanValueIssue } from "../../../api";
+import { isPossibleBindingAttributeValue } from "../../utils/is-binding-attribute-value";
+
+export function validateBooleanValue(
+  attribute: XMLAttribute,
+  model: UI5SemanticModel
+): InvalidBooleanValueIssue[] {
+  const actualAttributeValue = attribute.value;
+  const actualAttributeValueToken = attribute.syntax.value;
+  if (
+    actualAttributeValue === null ||
+    actualAttributeValueToken === undefined ||
+    isPossibleBindingAttributeValue(actualAttributeValue)
+  ) {
+    return [];
+  }
+
+  const ui5Property = getPropertyByAttributeKey(attribute, model);
+  const propType = ui5Property?.type;
+  if (propType?.kind !== "PrimitiveType" || propType.name !== "Boolean") {
+    return [];
+  }
+
+  const possibleBooleanValues = ["true", "false"];
+  if (
+    find(possibleBooleanValues, (_) => _ === actualAttributeValue) === undefined
+  ) {
+    const possibleValuesWithQuotes = map(
+      possibleBooleanValues,
+      (_) => `"${_}"`
+    );
+    return [
+      {
+        kind: "InvalidBooleanValue",
+        message: `Invalid value: ${
```

Comment:
should the message include the word "boolean"?
Would it make it easier to understand?

Reply:
Regarding this and the "one of" comment, we should consult with UA on the message

Reply:
I will change according to your suggestion and we'll consult UA later